src/bentoml/_internal/monitoring/otlp.py

```diff
@@ -27,7 +27,12 @@
 from .base import MonitorBase
 
 try:
-    from opentelemetry.exporter.otlp.proto.http._log_exporter import OTLPLogExporter
+    from opentelemetry.exporter.otlp.proto.grpc._log_exporter import (
```

Comment:
can you check whether the gRPC logs exporter requires any additional dependencies that we should include?